app/react/Forms/components/Geolocation.js

```diff
@@ -39,13 +45,21 @@ export default class Geolocation extends Component {
   }
 
   latChange(e) {
-    const { lon, label } = this.getInputValues();
-    this.onChange({ lat: parseFloat(e.target.value), lon, label });
+    let latitude = e.target.value ? parseFloat(e.target.value) : '';
+    latitude = latitude && latitude < -90 ? -90 : latitude;
```

Comment:
Apparently this is the wrong limit.  While the system will accept +90, it won't accept -90.

Negative latitude of -90 still breaks the rendering!  This is more complex than the positive value limiting, as anything above 90 should be 90, but we cannot use integers with the negative value, as limiting to -89 will leave a full degree of error (albeit ti would always be in the Antarctica!).

So, lets use 5 decimal points (about 1 meter worth of distance) and go with -89.99999 for anything above that value.

Reply:
Thank you Rafa, I change it now.